### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by Bharanidharan7708** [Bharanidharan7708/Hacktober2023#90]
I need someone to update the readme, I want to add a small tutorial on how to do the first PR for beginners. It needs to be below the contributor's section, make it look good and professional, and most importantly make it easy to understand, Thanks.

**Comment by Reene-Simon** [comment]
I would like to work on this issue. Could you please assign this issue to me? Please do add the hactoberfest accepted tag. Thanks.

**Comment by Bharanidharan7708** [comment]
Yes, go ahead and make a good one! Good luck.

**Comment by Reene-Simon** [comment]
@Bharanidharan7708 Yes sure. Just a request. Could you please add the Hacktoberfest accepted tag? Thanks!

**Comment by Bharanidharan7708** [comment]
@Reene-Simon Done

**Comment by Reene-Simon** [comment]
Yeah thanks. Will send a PR once done.

**Comment by Reene-Simon** [comment]
@Bharanidharan7708 this PR was there today morning as accepted on my Hacktoberfest portal but now it shows that the contribution is excluded as the repo itself has been excluded from Hacktoberfest. Is there anything you can do from your end? As creating a ReadMe for a project is a valid contribution.
